src/index.html

```diff
@@ -5,8 +5,323 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__container">
+          <li>
+            <a href="#" class="nav__list is-active">Apple</a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__list">Samsung</a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__list">Smartphones</a>
+          </li>
+
+          <li>
+            <a 
+              href="#" 
+              class="nav__list" 
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__list">Gadgets</a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__list">Tablets</a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__list">Photo</a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__list">Video</a>
+          </li>
+        </ul>   
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="product-info" data-qa="card">
+        <img 
+          src="./images/imac.jpeg" 
+          alt="Product" 
+          class="product-info__img"
+        >
+  
+        <h4 class="product-info__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h4>
```

Comment:
```suggestion
        <h4 class="product-info__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h4>
        
```
too long for one line